CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Contributing to ECMAScript
+There are a number of ways to contribute to ECMAScript. How to actually contribute depends on what you want to accomplish. In many cases, you may be required to execute a CLA with ECMA. See the final section named *Signing the CLA* for more information on this.
+
+Ecma TC39 accepts contributions from non-member individuals who have accepted the TC39 copyright and patent policies. Currently all ECMAScript related technical work is done by the TC39 RF TG (Royalty Free Task Group), for which the following IPR Policies apply:
+
+  * [Ecma International RF Patent Policy](http://www.ecma-international.org/memento/Policies/Ecma_Royalty-Free_Patent_Policy_Extension_Option.htm)
+  * [Ecma International Software Copyright Policy](http://www.ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.htm) ([PDF](http://www.ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.pdf))
+
+### Bug Reports & Bug Fixes
+File a bug or pull requests against the current text of ECMA262 in this repository. If the fix is trivial you may not need to sign the CLA. If your fix is involved, signing the CLA will be required.
+
+### Feature requests
+Feature requests for future versions of ECMAScript should not be made in this repository. To make a feature request, post to the es-discuss mailing list. Your goal will be to convince others that your proposal is a useful addition to the language and recruit TC39 members to help turn your request into a proposal and shepherd it into the language.
+
+### Proposals
+If you have a new proposal you want to get into the language, you first need a TC39 champion. Head to es-discuss to find one. If you already have a TC39 champion, send a pull reqeust to add a link to your proposal to the Current Proposals list.
+
+Integration of your proposal into this repository is accomplished by a pull request once the proposal has moved through the process document.
```

Comment:
I want to say: if you have a proposal ready to go and it needs integration, send a PR. If you want to make a proposal, start with es-discuss.